src/identity/IdentityProviderHttpHandler.ts

```diff
@@ -168,7 +168,9 @@ export class IdentityProviderHttpHandler extends HttpHandler {
     oidcInteraction?: Interaction): Promise<void> {
     if (request.method === 'GET') {
       // .ejs templates errors on undefined variables
```

Comment:
I think this comment can go indeed.

BTW (and this is asked somewhere else), the way around that would be `if (typeof myvar !== 'undefined')` but let's not go there. If common, I'd suggest to instead pass an object and then do `if (params.myvar)` where `params` is always defined.